config/src/genesis_config.rs

```diff
@@ -827,7 +832,7 @@ pub static G_HALLEY_BOOT_NODES: Lazy<Vec<MultiaddrWithPeerId>> = Lazy::new(|| {
 
 pub static G_HALLEY_CONFIG: Lazy<GenesisConfig> = Lazy::new(|| {
     // let stdlib_version = StdlibVersion::Latest;
-    let stdlib_version = StdlibVersion::Version(11);
+    let stdlib_version = StdlibVersion::Version(12);
```

Comment:
We use version 12 , not use Latest?

Reply:
Currently the latest stdlib version is 12.